tests/integration/components/yeti-table/general-test.js

```diff
@@ -643,8 +644,15 @@ module('Integration | Component | yeti-table (general)', function (hooks) {
   });
 
   test('yielded columns, visibleColumns, totalRows and visibleRows are correct', async function (assert) {
+    this.onVisibleRowsChanged = rows => {
+      set(this, 'savedVisibleRows', rows);
+    };
+    this.onFilteredRowsChanged = rows => {
+      set(this, 'savedRows', rows);
+    };
+
     await render(hbs`
-      <YetiTable @data={{this.data}} as |table|>
+      <YetiTable @data={{this.data}} @onVisibleRowsChanged={{this.onVisibleRowsChanged}} @onFilteredRowsChanged={{this.onFilteredRowsChanged}} as |table|>
```

Comment:
There's one case that I'd like to test:
what if the `@onVisibleRowsChanged` changes something that in turn makes `@onVisibleRowsChanged` to change again (just once, to avoid an infinite loop)?

I suspect ember will throw an error because you're changing something twice in the same runloop.
If that happens, we should call the action in the actions queue: https://api.emberjs.com/ember/3.19/functions/@ember%2Frunloop/scheduleOnce